app/models/billing/usage/product_catalog.rb

```diff
@@ -29,4 +29,13 @@ def free_products
   protected
 
   attr_reader :parent
+
+  def non_zero_included_prices
+    parent
+      .team
+      .billing_subscriptions
+      .active
+      .flat_map(&:included_prices)
+      .select { |included_price| included_price.quantity && included_price.quantity > 0 }
```

Comment:
I would have added a scope, but I am worried about repos like CF that have already cloned the starter repo. I.e. if I were to deploy a new scope, and upgrade CF, everything would break because we override the IncludedPrice model, and there would be no scope definition in there.